src/assets/CollectiblesController.ts

```diff
@@ -466,19 +469,32 @@ export class CollectiblesController extends BaseController<
    * @param address - Hex address of the collectible contract.
    * @param tokenId - The collectible identifier.
    * @param collectibleMetadata - Collectible optional information (name, image and description).
+   * @param detection - An object containing the users currently selected address and the chainId used to ensure detected collectibles are added to the correct account.
```

Comment:
This description seems wrong. `this.config.chainId` and `this.config.selectedAddress` are the users currently selected network and address. If that's what this was intended to be, we could just use the config.

Would this be more accurate?
```suggestion
   * @param detection - The chain ID and address of the currently selected network and account at the moment the collectible was detected.
```

Reply:
I guess this same description could be used for `addCollectibleContract` and `addCollectible` as well

Reply:
Changed in d7f56d0c93718842f59fb23002cafbf1d4e70162